chia/cmds/stop.py

```diff
@@ -20,6 +21,8 @@ async def async_stop(root_path: Path, config: Dict[str, Any], group: str, stop_d
         r = await daemon.exit()
         await daemon.close()
         if r.get("data", {}).get("success", False):
+            [print(f"{service}: Stopped") for service in r["data"]["services_stopped"]]
```

Comment:
it will throw an error if services_stopped is not present

Reply:
would it not always be there?

Reply:
cant be too safe, ill add a check

Reply:
safe then sorry, good point